charts/values.schema.json

```diff
@@ -14,6 +14,7 @@
                         "ContainerRuntime": {
                             "type": "string",
                             "default": "podman",
+                            "description": "The option needs to be determined based on the runtime environment of the current deployed cluster. If the cluster runtime is Docker, both options can be chosen. If the runtime is not Docker, Podman must be selected.",
```

Comment:
ContainerRuntime defines the tools used by the Agent to run and build containers. It’s important to note that "Docker" should only be selected when the container runtime on the node is Docker. For other container runtimes, please select "Podman".